### DISCUSSION TITLE
RT-DETR断点续训报错

### DISCUSSION THREAD

**Discussion by dxm620** [ultralytics/ultralytics#13406] [Q&A]
RT-DETR的训练意外中断，使用官方文档的断点继续训练方法后，出现报错AttributeError: 'RTDETRDecoder' object has no attribute stride，如何解决呢？

**Comment by pderrenger** [comment]
您好！看起来您在尝试使用断点续训功能时遇到了问题。这个错误提示可能是由于模型定义与权重不匹配导致的。请确保您加载的预训练模型与您的代码中定义的模型架构完全一致。

如果问题仍然存在，请尝试清除任何缓存的模型文件，并重新下载或重新加载模型。如果这些步骤不能解决问题，您可以在这里提供更多的错误日志和代码片段，我们将进一步帮助您解决问题。感谢您使用Ultralytics YOLO！🚀
